### DISCUSSION TITLE
Breaking change in release 0.1.3

### DISCUSSION THREAD

**Discussion by nashysolutions** [pointfreeco/vapor-routing#28] [General]
![Screenshot 2023-04-16 at 22 20 31](https://user-images.githubusercontent.com/64097812/232343115-59f44263-c4fe-4b66-bb68-432044e08e34.jpg)

```
dependencies: [
        .package(url: "https://github.com/vapor/vapor.git", from: "4.0.0"),
        .package(url: "https://github.com/vapor/fluent.git", from: "4.0.0"),
        .package(url: "https://github.com/vapor/fluent-postgres-driver.git", from: "2.0.0"),
        .package(url: "https://github.com/vapor/leaf.git", from: "4.0.0"),
        .package(url: "https://github.com/pointfreeco/vapor-routing", from: "0.1.0")
    ],
```

Forcing previous release fixes my issue

```
 .package(url: "https://github.com/pointfreeco/vapor-routing", esact: "0.1.2")
```

**Comment by mbrandonw** [comment]
Hi @nashysolutions, this library is based on [swift-parsing](https://github.com/pointfreeco/swift-parsing/) which was forced to undergo some breaking changes due to changes in Swift regarding result builders. We had along [discussion](https://github.com/pointfreeco/swift-parsing/discussions/290) about that over in that repo.

I am going to convert this to a discussion as it was not something we could work around unfortunately. Please feel free to ask questions about how you might migrate your code to the newest library if it is not already covered in the previous discussion.
